.github/actions/collect-test-artifacts/action.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Collect test artifacts
+
+description: Collects test outputs and uploads them as artifact
+
+inputs:
+  name:
+    description: 'Name of the test artifact.'
+    required: true
+  sed-bin:
```

Comment:
:x: This appears to be left-over?

Reply:
Sorry I missed to delete that, I tried different approaches and you are right this is a left over